Cannot find module 'core-js/modules/es6.set'

The compiled code depends on `core-js` but it is not in `package.json`.

```
Error: Cannot find module 'core-js/modules/es6.set'
```

Actually I think we don't need this polyfill since we usually use this plugin with Node.js.

--- a/.babelrc
+++ b/.babelrc
@@ -4,8 +4,7 @@
       "targets": {
         "node": 4
       },
-      "loose": true,
-      "useBuiltIns": "usage"
+      "loose": true
     }]
   ]
 }

